libsentrykube/quickpatch.py

```diff
@@ -1,15 +1,122 @@
-from typing import Sequence, Mapping
+import os
+from typing import MutableMapping, Optional, Sequence, Union
+import yaml
+import jsonpatch
+from jinja2 import Template
+
+from libsentrykube.service import (
+    get_service_path,
+    get_service_value_overrides_file_path,
+)
+from jsonschema import validate, ValidationError
+
+
+def find_patch_files(service: str, patch: str) -> Optional[str]:
```

Comment:
I would change this to `find_patch_file` instead of `find_patch_files` since there can only be 1 patch file.

What do you think about changing the return type to `Optional[Path]` instead of `Optional[str]`. Does it make it easier to reason about what the function does i.e. finding the patch file path?